backend-api/src/functions/asyncTxmaEvent/validateRequestBody/validateRequestBody.test.ts

```diff
@@ -97,20 +97,26 @@ describe("Validate request body", () => {
   });
 
   describe("Given body is valid", () => {
-    it("Returns a successResult with a valid parsed body as value", () => {
-      const result = validateRequestBody(
-        JSON.stringify({
-          sessionId: mockSessionId,
-          eventName: "DCMAW_ASYNC_HYBRID_BILLING_STARTED",
-        }),
-      );
+    describe.each([
+      { eventName: "DCMAW_ASYNC_HYBRID_BILLING_STARTED" },
+      { eventName: "DCMAW_ASYNC_READID_NFC_BILLING_STARTED" },
+      { eventName: "DCMAW_ASYNC_IPROOV_BILLING_STARTED" },
+    ])("$eventName", ({ eventName }) => {
+      it(`Returns a successResult for ${eventName} with a valid parsed body as value`, () => {
```

Comment:
if we do the above suggestion, we can then remove it from the it block
```suggestion
      it(`Returns a successResult with a valid parsed body as the value`, () => {
```